staging/src/kubevirt.io/client-go/api/v1/defaults.go

```diff
@@ -116,6 +117,10 @@ func SetDefaults_VirtualMachineInstance(obj *VirtualMachineInstance) {
 		obj.Spec.Domain.Features = &Features{}
 	}
 
+	if obj.Spec.SchedulerName == "" {
+		obj.Spec.SchedulerName = v1.DefaultSchedulerName
```

Comment:
So far we didn't set the schedulerName on the pod spec, perhaps we should continue and set it only if a specific scheduler name was provided? Can't there be an admission controller or something else that would set a non-default scheduler name if nothing is set?

Reply:
Good point. Fixed.